alternator/executor.cc

```diff
@@ -2733,22 +2733,35 @@ static std::optional<rjson::value> action_result(
     }, action._action);
 }
 
+}
+
 // Print an attribute_path_map_node<action> as the list of paths it contains:
-static std::ostream& operator<<(std::ostream& out, const attribute_path_map_node<parsed::update_expression::action>& h) {
+template <> struct fmt::formatter<alternator::attribute_path_map_node<alternator::parsed::update_expression::action>> {
+    constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+    // this function recusively call into itself, so we have to forward declare it.
```

Comment:
spelling mistake (caught by the bot): recusively.

Reply:
thanks. that's exactly what i am expecting from the workflow!